From 4724d58a0a378e03067230e173d310b547b998b6 Mon Sep 17 00:00:00 2001
From: Vinay Prasad Kompella <vinay.kompella@intel.com>
Date: Tue, 17 May 2022 10:44:31 +0000
Subject: [PATCH] Expose x86 sources for usage in libc

Static linking libcpu_features with libc has cyclic dependency issues. Hence, the choice to expose sources.

Test: make
Signed-off-by: Vinay Prasad Kompella <vinay.kompella@intel.com>
Change-Id: Ice3d874507d30d4215141889830e08df56b05543
---
 Android.bp | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Android.bp b/Android.bp
index 7f70060..d92342d 100644
--- a/Android.bp
+++ b/Android.bp
@@ -89,6 +89,13 @@ cc_library {
     ],
 }
 
+filegroup {
+    name: "libcpu_features_x86_sources",
+    srcs: [
+        "src/cpuinfo_x86.c",
+    ],
+}
+
 cc_library {
     name: "libcpu_features",
     defaults: [
-- 
2.17.1

